src/options.h

```diff
@@ -52,4 +52,6 @@ class Options {
   bool perform_geometric_verification_ = false;
 };
 
+enum ReconstructionStatus { NOT_BUILT, BUILT, LOADED_INTO_MEMORY };
```

Comment:
I would specify the value of every enum member.
`enum ReconstructionStatus { NOT_BUILT = 0, ... };`
that allows us to remove reduntand StatusFromInt method

more: http://en.cppreference.com/w/cpp/language/enum